app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
       android:allowBackup="true"
       android:hardwareAccelerated="true"
       android:icon="@mipmap/kiwix_icon"
+      android:roundIcon="@mipmap/kiwix_icon_round"
```

Comment:
Just move this above `android:supportsRtl="true"`. This is to preserve our code formatting. :)